### ISSUE TITLE
[1.1.4] ImagePath not working anymore

### ISSUE THREAD

**Issue by alexselea** [RaiMan/SikuliX1#101]
I'm using sikuli for my automated tests in java with Selenium. I'm packing all within a jar and it worked up until 2 weeks ago. When I try to add the imagePath, the function returns true, but sikuli is unable to find any image. When I first encountered this error I put a version that was working in my pom.xml (20190308.150216-122) and it got solved. Now I'm trying to set them up on another computer which does not have that sikulixapi version cached so I cannot use it, so I'm using 1.1.4-SNAPSHOT in pom.xml instead.

After setting up ImagePath with ```ImagePath.add(Main.class.getCanonicalName() + "/images")``` I am printing the ImagePath with ```ImagePath.getBundlePath()``` and it returns null. 

For reference, my project has ```src/main/java/Main.java``` and ```src/main/java/resources/images```. In the final jar there is ```main/java/Main.class``` and the ```images``` folder in the root of the jar in which all the images are stored. The ImagePath worked up until 10.03.2019 as far as I'm aware.

Thank you

**Comment by RaiMan** [comment]
The ImagePath is a list, with the bundlePath at [0], which by default is null.
add() adds path entries from [1] upwards.

As long as you do not use ImagePath.setBundlePath(somePath), getBundlePath() will return null.

**Comment by RaiMan** [comment]
I tested your case with ``add()`` and ``setBundlePath()``:
both worked as expected.

**Comment by RaiMan** [comment]
no feedback

**Comment by alexselea** [comment]
Hi, it was still not working for me but I had no time to try again to use the latest version, I am using that 20190308 that maven cached that time. The same code finds the images in the old version, and doesn't find them in the new one.

**Comment by RaiMan** [comment]
As mentioned works with the latest build
[info] Version: 1.1.4-SNAPSHOT-#266-2019-04-03_08:08

tested with
```
    String name = MavenTest.class.getCanonicalName();
    ImagePath.add(name + "/Images");
    Screen scr = new Screen();
    scr.hover("img");
```

**Comment by alexselea** [comment]
The code is exactly the same, I'm packing everything in a jar and when I run it I'm getting a null pointer exception at ```ork.sikuli.script.ImagePath.getBundlePath(ImagePath.java:609```

**Comment by RaiMan** [comment]
please paste the relevant code snippet.

**Comment by RaiMan** [comment]
Ok, sorry for being so blind (did not test getBundlePath(), because it is not relevant in your situation).

It is now fixed (build #267).

... but getBundlePath() will not return any valid content until you have used setBundlePath() or put a default path to Settings.bundlePath().

**Comment by march-lee** [comment]
What are the rules of use ImagePath?my project can't find image,exception:FindField.
Screen screen = new Screen();
        String name = SikuliTest.class.getCanonicalName();
        String[] path = ImagePath.get();
        for (int i = 0; i <path.length ; i++) {
            System.out.println(path[i]);
        }
        System.out.println(name);

        ImagePath.add(name + "/imagePackge");
My project pathis :com.ccb.SikuliTest,,com.ccb.imagePackge
Error message:[error] ImagePath: add: not valid: com.ccb.SikuliTest/imagePackge
